app/assets/javascripts/short-form/templates/c4-household-public-housing.html.slim

```diff
@@ -1,14 +1,22 @@
 .app-card.form-card
   short-form-header
-    h2.app-card_question
-      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate: youOrHouseholdTranslateVariable() }}
+    h2.app-card_question ng-if="householdMembers.length == 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate }}
+    h2.app-card_question ng-if="householdMembers.length > 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_HOUSEHOLD' | translate }}
     p.form-note.margin-bottom
       | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.P1' | translate}}
 
   .app-inner.inset
     .form-group ng-class="{ error: inputInvalid('STUB_householdPublicHousing') }"
       p.form-note.margin-bottom translate="LABEL.PLEASE_SELECT_ONE"
-      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate}}"
+      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate}}" ng-if="householdMembers.length == 0"
+        radio-block-item value='Yes' user='application' name='STUB_householdPublicHousing' ng-required='true' on-change="publicHousingYes"
```

Comment:
@kaptron Ah I see, didn't know you could point the area label to an ID of another element. Cool! I'll do that.